.github/workflows/cli.yaml

```diff
@@ -0,0 +1,74 @@
+name: CLI
+on:
+  # Run this workflow every time a new commit pushed to upstream/fork repository.
+  # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
+  push:
+  pull_request:
+jobs:
+  init:
+    name: init
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        # Here support the latest three minor releases of Kubernetes, this can be considered to be roughly
+        # the same as the End of Life of the Kubernetes release: https://kubernetes.io/releases/
+        # Please remember to update the CI Schedule Workflow when we add a new version.
+        k8s: [ v1.24.2, v1.25.0, v1.26.0 ]
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          # We need to guess version via git tags.
+          fetch-depth: 0
+      - name: install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.5
+      - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.17.0"
+      - name: build cli
+        run: |
+          set -x
+
+          # tar the newest crds
+          cd  charts/karmada/
+          cp -r _crds crds
+          tar -zcvf ../../crds.tar.gz crds
+          cd -
+
+          # prepare karmadactl
+          make karmadactl
+          cp _output/bin/linux/amd64/karmadactl /usr/local/bin/
+      - name: run karmadactl init test
+        run: |
+          set -x
+
+          export CLUSTER_VERSION=kindest/node:${{ matrix.k8s }}
+
+          # create host cluster
+          hack/create-cluster.sh host $HOME/.kube/host.config
+          kubectl wait --for=condition=Ready nodes --all --timeout=800s --kubeconfig=${HOME}/.kube/host.config
+
+          # init Karmada control plane
+          karmadactl init --kubeconfig=${HOME}/.kube/host.config \
+            --karmada-data=${HOME}/karmada \
+            --karmada-pki=${HOME}/karmada/pki \
+            --crds='./crds.tar.gz'
+
+          # create member cluster
+          hack/create-cluster.sh member1 $HOME/.kube/member1.config
+          hack/create-cluster.sh member2 $HOME/.kube/member2.config
```

Comment:
I think we can create the `3` clusters in parallel so that the block time could be significantly reduced.
Similar to what we did at `hack/local-up-karmada.sh`: 
https://github.com/karmada-io/karmada/blob/83c7862289ac3884985cee2a10f023ffb8f925b2/hack/local-up-karmada.sh#L84-L86